submissions/ses3332002/html-css-popup/build/style.css

```
@@ -0,0 +1,144 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+  background-color: white;
+}
+body a {
+  text-decoration: none;
+  color: unset;
+}
+body a:focus {
+  outline-style: solid;
+  outline-offset: 4px;
+  outline-color: #6098f3;
+  outline-width: 2px;
+}
+body input[type=checkbox] {
+  position: absolute;
+  -webkit-appearance: none;
+  appearance: none;
+}
+
+header {
+  text-align: right;
+}
+header .user_menu {
+  display: inline-flex;
+  align-items: center;
+  justify-content: space-around;
+  font-size: 0;
+}
+header .user_menu .user_menu__element {
```

You don't need to use such `heavy` selectors. Just using only class selector will be ok in these cases
```suggestion
.user_menu__element {
```

---

I agree about "heavy" selectors, but it's not my fault :) it's because of nested style structure in Sass. 
If that is a problem I will not use nested selectors in Sass. Do I have to remake selectors in this task?
About keyboard navigation: your can jump to items in popup menu after last item in "top level" menu. 
There was not any instructions about navigation order in the task, so I did not arrange any tab order, just convinced that every element is able via keyboard. What tab order is the right one?

---

Ok. Got you. But as I know it's not the best practice to use selectors with the deep nesting in Sass too. 
It's up to you. I think it works for me for this task. Please, don't forget to check the builds of styles in the future.
Hm... About the requirements. Let me ask our layout guru Sasha.
@A-Ostrovnyy, could you please check the preview of this PR?